crates/rattler_conda_types/src/utils/serde.rs

```diff
@@ -152,3 +155,45 @@ pub(crate) fn sort_map_alphabetically<T: Serialize, S: serde::Serializer>(
         .collect::<BTreeMap<_, _>>()
         .serialize(serializer)
 }
+
+/// A helper to serialize and deserialize `track_features` in repodata. Track
+/// features are expected to be a comma seperated list. However, in the past we
```

Comment:
This is then wrong?

Reply:
Yes thats before I learned that it was crazier then I thought.. I changed it.